app/main/views/service_updates.py

```diff
@@ -50,37 +52,60 @@ def service_status_update_audits(day=None, page=1):
     )
 
 
-@main.route('/service-updates', methods=['GET'])
+_service_update_audits_event_limit = 1000
```

Comment:
I guess the choice is either make it uppercase, remove the underscore and imply it's a dickable-with ... thing ... or imply that it's not.